.github/workflows/publish.yml

```diff
@@ -1,18 +1,23 @@
 name: publish
 on:
-    release:
-        types: [created]
+    - push
```

Comment:
Is there ever a case where we would want to **manually** trigger publish? If so, is there another action we can use to trigger that? Possibly a tag?

Reply:
I don't know when we'd want to manually trigger this `publish` workflow, but we can by listening to the [`workflow_dispatch` event](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#configuring-a-workflow-to-run-manually). That allows the `publish` workflow to be [run manually](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow).

Do  you think we should add that now?

Reply:
Eh, if we don't have a reason to yet, then I don't think it's worth spending any time on it. I think knowing that we have a path forward is enough, in case we ever need it.

Reply:
Okay in that case, we can at least document the idea in `RELEASE.md`.